Assembler.Core/Entities/BaseAssembledMessage.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Assembler.Core.Entities
+{
+    public abstract class BaseAssembledMessage : BaseMessage
+    {
+        protected BaseAssembledMessage()
```

Comment:
Unnecessary

Reply:
It's just for classification

Reply:
Didn't really understand what you mean

Reply:
It's the same as BaseMessage but I kept it there as a classification, we classify what inherits from it as an assembled message

Reply:
We might not be on the same page - I specifically meant the empty c'tor is unnecessary, the class itself is fine

Reply:
You are right

Reply:
Just a heads-up - you still didn't update this

Reply:
I did
![image](https://user-images.githubusercontent.com/3995473/77670312-b28f5b00-6f8e-11ea-97aa-d5939c58c70a.png)